src/components/NavBar/menu-list.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Tab } from "@headlessui/react";
+
+import { NavPath } from "@/types";
+
+interface Props {
+  items: Array<NavPath>;
+}
+
+export default function MenuList({ items }: Readonly<Props>) {
+  const router = useRouter();
+
+  return (
+    <Tab.Group>
+      <Tab.List className="flex pl-1">
+        {items.map((item) => (
+          <Tab
+            key={item.item}
+            className={`w-[160px] h-[60px] rounded-lg font-medium outline-none text-black hover:bg-slate-100 hover:text-primary-700 ${
```

Comment:
In this case, I recommend not setting a height restriction on the tab, since the navbar is already restricted. If you need spacing, use padding or margins